src/one_dragon/base/notify/push.py

```
@@ -566,28 +566,124 @@ def send_mpnews(self, title, message, media_id, touser="@all"):
+            import io
+            from PIL import Image
+            image.seek(0)
+            pil_img = Image.open(image)
+            # 优先压缩为 JPEG
+            with io.BytesIO() as output:
+                pil_img = pil_img.convert('RGB')
+                pil_img.save(output, format='JPEG', quality=85)
+                img_bytes = output.getvalue()
+                if len(img_bytes) <= 2 * 1024 * 1024:
+                    return img_bytes, 'jpeg'
+            # 如果 JPEG 超过 2MB，则尝试 PNG
```

Ruff (0.12.2)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)